OsmIntegrator/Tools/TokenHelper.cs

```diff
@@ -40,11 +40,11 @@ public TokenHelper(IConfiguration configuration)
             {
                 new Claim(JwtRegisteredClaimNames.Sub, userId),
                 new Claim(JwtRegisteredClaimNames.Jti, Guid.NewGuid().ToString()),
-                new Claim(ClaimTypes.NameIdentifier, user.Id.ToString()),
+                new Claim("user_uuid", user.Id.ToString()),
```

Comment:
Better will be `userId` because in the future we'll replace `guid` with long (int). 
Also, I think it's a json format so camel case is more preferred.

Reply:
done